andela/resly
# [HotFix #119262913] Fix the bug with the cancelled reservations

#### What does this PR do?

This PR is a hot-fix for the bug found in the cancelled reservation feature.
#### Description of Task to be completed?

The cancelled reservations should be tied to the current user and not all the users.
Also once a reservation is cancelled the table should be released.
#### How should this be manually tested?

> Login as a diner and make a reservation by completing payment.
> Cancel the reservation and view the page of correct cancelled reservations.
> Go to the restaurants search page and select the table.

The table should no longer be booked.
#### Any background context you want to provide?

A cancelled reservation should release its table so that other users can make reservations on such table.
#### What are the relevant pivotal tracker stories?

This story can be viewed on PT [here](https://www.pivotaltracker.com/projects/1415814/stories/119262913).
### Screenshots (if appropriate)

![screen shot 2016-05-10 at 11 28 23 am](https://cloud.githubusercontent.com/assets/17028608/15143636/a2945cda-16a2-11e6-974c-6ad9b9c8c61c.png)
![screen shot 2016-05-10 at 11 29 10 am](https://cloud.githubusercontent.com/assets/17028608/15143637/a294af5a-16a2-11e6-9a9d-2587d7e2558b.png)
![screen shot 2016-05-10 at 11 29 45 am](https://cloud.githubusercontent.com/assets/17028608/15143638/a2b4d294-16a2-11e6-99ed-b4f3295322be.png)
#### Questions:

There are no questions at this moment.
